users/tests/test_user_login_logout.py

```diff
@@ -0,0 +1,36 @@
+from rest_framework.test import APITestCase, APIRequestFactory
+from rest_framework import status
+from users.models import User
+from users.tests.factories import UserFactory
+import faker
+
+class LoginTests(APITestCase):
+    def setUp(self):
+        self.user = UserFactory()
+        self.password = faker.Faker().password()
+        self.user.set_password(self.password)
+        self.user.save()
+
+    def test_sign_in_success(self):
+        data=({'email': self.user.email, 'password': self.password})
+        response = self.client.post('/dj-rest-auth/login/', data)
```

Comment:
Maybe you can add a `self.url = '/dj-rest-auth/login/'` to the set up and use that instead of adding the url in each test. Also, check [reverse](https://docs.djangoproject.com/en/3.1/ref/urlresolvers/#reverse). It is usually used when managing urls, in this case it would be `self.url = reverse('rest_login')`. This is a special case because the login endpoint wasn't defined by you, but you can see the name [here](https://github.com/jazzband/dj-rest-auth/blob/master/dj_rest_auth/urls.py#L12).